data/remnant/remnant 1 introduction.txt

```diff
@@ -127,42 +127,59 @@ mission "First Contact: Remnant"
 
 
 # Compatibility patch for those who made contact with the Remnant prior to v0.9.9
-mission "Remnant Blood Test Patch"
+# This will also catch everyone who completed this mission prior to the correction in 0.9.15 to give them "remnant: blood test pure"
+mission "Remnant: Blood Test Patch"
 	landing
 	invisible
 	to offer
 		has "event: ember waste label"
 	on offer
-		set "remnant blood test pure"
+		set "remnant blood test pure" # This line is depreciated and will be removed in future versions. Test for "remnant: blood test pure" instead.
+		set "remnant: blood test pure"
 		fail
 
 
 
 # Changes Remnant hails once contacted and submitted to blood test
-mission "Remnant Hails Change"
+mission "Remnant: Hails Change"
 	landing
 	invisible
 	to offer
-		has "remnant blood test pure"
+		has "remnant: blood test pure"
 	on offer
-		event "remnant change hail"
+		event "remnant: change hail"
 		fail
 
 
 
-mission "lost in ember waste"
+mission "Remnant: lost in ember waste"
 	landing
 	invisible
 	source
 		government "Remnant"
 	to offer
-		has "remnant blood test pure"
+		has "remnant: blood test pure"
 		not "license: Remnant"
 	on offer
 		require "Quantum Keystone" 0
 		require "Quantum Key Stone" 0
 		require "Jump Drive" 0
-		event "lost in ember waste" 14
+		event "remnant: lost in ember waste" 14
+		fail
+
+
+
+mission "Remnant: lost in ember waste compatibility"
+	landing
+	invisible
+	source
+		government "Remnant"
+	to offer
+		has "lost in ember waste: declined"
+	on offer
+		set "Remnant: Lost in Ember Waste: offered"
+		set "Remnant: Lost in Ember Waste: declined"
```

Comment:
The mission name still has lowercase "lost in ember waste", yet this compatibility mission uses capitalized words.

Reply:
Fixed. Thanks for pointing that out.